portal/oauth/README.md

```diff
@@ -10,7 +10,7 @@ Your app may need to access items that are only shared with authorized users. Fo
 
 ## How to use the sample
 
-When you run the sample, the app will load a web map which contains premium content. You will be challenged for an ArcGIS Online login to view the private layers. Enter a user name and password for an ArcGIS Online named user account (such as your ArcGIS for Developers account). If you authenticate successfully, the traffic layer will display, otherwise the map will contain only the public basemap layer.
+When you run the sample, the app will load a web map which contains premium content. You will be challenged for an ArcGIS Online login to view the private layers. Enter a user name and password for an ArcGIS Online named user account (such as your ArcGIS for Developers account). If you authenticate successfully, the traffic layer will display, otherwise the map will contain only the public basemap style layer.
```

Comment:
is the BasemapStyle.ARCGIS_STREETS basemap still classed as a public basemap layer? I wasn't sure if it's accessible without an API key and therefore if it's ok to still refer to it as public. To be on the safe side we could just go with:

> otherwise the map will contain only the basemap style layer

Reply:
Nice spot! I think that sounds right.